### DISCUSSION TITLE
Uvas Explosivas

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#532] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/uvas-explosivas/

**Comment by the-coffee-monk** [comment]
*"Infelizmente não são as Esferas do Dragão"* :rofl:

**Comment by brunoaugusto** [comment]
> "Atenção! Muito Cuidado caso tente realizar o experimento com seu próprio micro-ondas!"

Preciso… resistir… ao impulso… besteirol :grin:

**Comment by najarin-the-loremaster** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/curiosidades/uvas-explosivas/#comment_1174175802>):

> Preciso… resistir… ao impulso… besteirol 😁

Contanto que não tente aquecer cachos inteiros, provavelmente, o pior que pode acontecer é fazer bastante sujeira. Mas ainda assim, por sua conta e risco.
